card/mod/core/set/all/initialize.rb

```diff
@@ -11,6 +11,7 @@ def new args={}, _options={}
 end
 
 def initialize args={}
+  args["name"]= Card.compose_mark(args["name"]) if args["name"].is_a?(Array)
   args["name"] = args["name"].to_s
```

Comment:
I think this one sent the complexity over the hound's threshold, and that one passes the sniff test.  separate out `initial_name`?